dna/problems.py

```diff
@@ -129,6 +96,90 @@ def predict(self, data, model, model_config, *, verbose=False, model_output_dir=
         return predictions_by_group, predict_time
 
 
+class RegressionProblem(PredictByGroupProblemBase):
```

Comment:
Code is harder to review when it is both changed and moved to another location. I cannot easily see what changes were made. Can you move this class back to its original location?

Reply:
The regression problem has been moved back. It's not on the exact same line because of the import at the top and the removing of the predict method of Problem base. That code was moved to the predict_all method of RegressionProblem. Let me know if you'd like ProblemBase to at least have a predict method that raises a NotImplementedException.